app/src/main/java/com/infomaniak/mail/data/api/ApiRoutes.kt

```diff
@@ -69,6 +69,18 @@ object ApiRoutes {
         return "${message(mailboxUuid, folderId, shortUid)}/attachmentsArchive"
     }
 
+    fun search(
+        mailboxUuid: String,
+        folderId: String,
+        filters: String,
+        offset: Int = 0,
+        isDraftFolder: Boolean = false,
+    ): String {
+        val offset = "offset=$offset"
```

Comment:
`offset` is already used, we might as well put this string directly in the return